### ISSUE TITLE
update manager "description" and "changelog" tabs title text color

### ISSUE THREAD

**Issue by Vahan86** [linuxmint/mint-y-theme#47]
The readability of the title text of "description" and "changelog" tabs from update manager in Mint-Y theme needs improvement since now text color is palish.

**Comment by JosephMcc** [comment]
Not sure I agree. That really has nothing to do with mintupdate.

That is how it looks in all gtk3 applications.

**Comment by Vahan86** [comment]
Sorry @JosephMcc, but I did not catch the point of what you said.

As you can see in the attached screenshot, in Mint-X theme the title text is more readable than in Mint-Y theme.

![6](https://cloud.githubusercontent.com/assets/12819058/19833173/b95974da-9e49-11e6-8feb-060b49b92fca.png)

**Comment by JosephMcc** [comment]
I was just saying it isn't an issue with mintupdate. It looks the same in all gtk3 notebooks. The preferences dialog in xed for example.

**Comment by Vahan86** [comment]
I agree with you that this is not a mintupdate related issue, that's why I reported this issue in mint-y-theme repository.
And why I mentioned in particular mintupdate, because I guess that after implementing new style tabs for xed for example this issue will be automatically solved, but I wasn't sure that the new style tabs will be implemented for mintupdate as well.
